cerbero/build/fridge.py

```diff
@@ -85,6 +85,16 @@ def __init__(self, remote, username='', password=''):
     def __str__(self):
         return 'remote \'{}\', username \'{}\', password \'{}\''.format(self.remote, self.username, self.password)
 
+    @classmethod
+    def fromNetRc(cls, remote):
```

Comment:
nitpick: snake case, as the rest of the code here

Reply:
done